VERSION

```diff
@@ -1 +1 @@
-0.35.4
+0.35.5
```

Comment:
Let's hold off here for the moment. Technically this change includes the script tag so, for the sake of simplicity, I think I'm gonna make the suggested changes to the `script` and make a `0.36.0` release.

Reply:
Err. Maybe we should release this but just not announce the `script`.

Reply:
I'll just pull the version bump out of this PR, and we can create a 0.36.0 release separately.